cmake/projects/zstd/hunter.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 Niall Douglas https://www.nedproductions.biz/
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_source_subdir)
+
+hunter_add_version(
+    PACKAGE_NAME
+    zstd
+    VERSION
+    1.4.5-d73e2fb-p0
+    URL
+    "https://github.com/facebook/zstd/tarball/d73e2fb465c9289f5a6871cef1fe48de917a5cc2"
```

Comment:
You can change this to `https://github.com/facebook/zstd/archive/d73e2fb465c9289f5a6871cef1fe48de917a5cc2.tar.gz`. Make sure to update the SHA1 as well!